### DISCUSSION TITLE
Does silk.net bind to imgui's main branch or docking branch?

### DISCUSSION THREAD

**Discussion by BreyerW** [dotnet/Silk.NET#546] [Q&A] [answered]
Im evaluating various gui solutions and i need docking capability (or at least building blocks for it). I see that silk.net provides imgui bindings along various other graphic api and assimp binding which is nice but i would like to clarify if imgui is based on main or docking branch (docking branch is updated along main branch btw it wasnt merged mostly due to sdl2 not behaving nicely in multiviewport scenarios if memory serves right but glfw should be largerly bugfree)

**Comment by Perksey** [comment, marked as answer]
We don't bind to ImGui at all. We delegate this responsibility to the ImGui.NET library. If you want the answer, you'd need to ask them.

**Reply by BreyerW** [reply]
Oh i see thanks for the answer

**Reply by BreyerW** [reply]
Checked and as of 25 mar 2021 imgui.net has docking branch if anyone had similar question
